docs/guides/dpdk-guide.md

```diff
@@ -95,7 +97,7 @@ certificates, see the [security_guide](https://github.com/ipdk-io/networking-rec
 
 ```bash
 cd $IPDK_RECIPE
```

Comment:
"cd $IPDK_RECIPE" this step may not be required now.

Reply:
Good catch. thanks